kythe/typescript/indexer.ts

```diff
@@ -1241,14 +1241,25 @@ class Visitor {
           console.error(`TODO: export ${exp.name} has no symbol`);
           continue;
         }
-        // TODO: import a type, not just a value.
         const remoteSym = this.typeChecker.getAliasedSymbol(localSym);
+        const anchor = this.newAnchor(exp.name);
         const kExport = this.host.getSymbolName(remoteSym, TSNamespace.VALUE);
-        this.emitEdge(this.newAnchor(exp.name), EdgeKind.REF, kExport);
+        const kExportType =
+            this.host.getSymbolName(remoteSym, TSNamespace.TYPE);
+        this.emitEdge(anchor, EdgeKind.REF, kExport);
+        const flags =remoteSym.flags;
+        const isAlsoType = (flags & ts.SymbolFlags.Interface) ||
+            (flags & ts.SymbolFlags.Class) || (flags & ts.SymbolFlags.Enum);
```

Comment:
I think this misses some cases of types, such as type aliases.

I think you want just
`(flags & ts.SymbolFlags.Type) !== 0`

Reply:
Thanks! That's much better. Also added reexporting of type to the test.